### ISSUE TITLE
Iphone 5s

### ISSUE THREAD

**Issue by kris0725pl** [foxlet/bootra1n#92]
Not working...error -1 sync

I have problem with load exploit...i have 12.8 iOS

**Comment by DEALWITHIT199** [comment]
Are you booting to the USB Drive? or Error of Jailbreaking?

**Comment by kris0725pl** [comment]
boot from usb...i have 5s with icloud password....i got this phone from friend. firmware 12.4.8.

i cant delete this icloud account.

I cant load to this device jailbreak firmware because i got error in 3utools SHSS

**Comment by DEALWITHIT199** [comment]
> boot from usb...i have 5s with icloud password....i got this phone from friend. firmware 12.4.8.
> 
> i cant delete this icloud account.
> 
> I cant load to this device jailbreak firmware because i got error in 3utools SHSS

I'm confused. You need to delete the iCloud Account of the phone first????

**Comment by DEALWITHIT199** [comment]
> boot from usb...i have 5s with icloud password....i got this phone from friend. firmware 12.4.8.
> 
> i cant delete this icloud account.
> 
> I cant load to this device jailbreak firmware because i got error in 3utools SHSS

Wait... So your trying to Jailbreak your iPhone 5s?

**Comment by kris0725pl** [comment]
i cant jailbreak....i have fuck...ng password icloud.

**Comment by DEALWITHIT199** [comment]
> i cant jailbreak....i have fuck...ng password icloud.

So your trying to remove the iCloud Account?

**Comment by kris0725pl** [comment]
yes...but no luck.

**Comment by DEALWITHIT199** [comment]
> yes...but no luck.

Yes! You can Remove the iCloud Account

**Comment by kris0725pl** [comment]
i try many programs...not working. Any suggestion?

**Comment by DEALWITHIT199** [comment]
1. You need to Install AnyUnlock to this page [https://www.imobie.com/anyunlock/iphone-password-unlocker.htm](url)
2. then install the setup. after the setup is fully installed open the Program
3. after opening the program you will see alot of features, one of them is "Unlock Apple ID" click on that and wait
4. when you see a new window opened, just follow the instructions provided there.

**Comment by DEALWITHIT199** [comment]
Forgot to ask. if your going to Jailbreak your iPhone 5s what version of Bootra1n are you going to use? and what version is your iOS?

**Comment by kris0725pl** [comment]
now i downgrade from 12.4.8. to 12.4.6 ios firmware using u3tools. I use bootrain bootra1n-i686-0.10.2-20200526.iso  Stuck on load exploit.  I try now with 12.4.6...maybe done.

**Comment by DEALWITHIT199** [comment]
umm why downgrade??

**Comment by kris0725pl** [comment]
maybe bootrain working on this version? on 12.4.8 not working. My screen from u3tools https://ibb.co/fCSvwtX

**Comment by DEALWITHIT199** [comment]
waittt don't downgrade yet!

**Comment by DEALWITHIT199** [comment]
Can you update your iPhone 5s to iOS 13.7?

**Comment by kris0725pl** [comment]
no....i have only high firmware 12.4.8 (flash) and 12.4.7 (jailbreak). Now i try again with botrain usb and i stuck on "Right before trigger (this is the real bug setup)....on this checkrain stop working...i waiting 10 minutes and nothing happen.

**Comment by DEALWITHIT199** [comment]
can you show me the error of bootra1n your trying to do?

**Comment by kris0725pl** [comment]
no problem...wait 5 minutes..i downloading x64 version live cd....i take screenshot my phone huawei...

**Comment by kris0725pl** [comment]
on this progress nothing happen....waiting 10 minutes and nothing.... https://ibb.co/KNCZ4M7

**Comment by DEALWITHIT199** [comment]
ok try doing this:

1. Use a High Quality Cable
2. Turn OFF Passcode
3. Put the USB Boot Drive on Slot on 2.0 not 3.0
4. Turn OFF wifi or turn on Airplane Mode with wifi OFF (optional)

**Comment by DEALWITHIT199** [comment]
If still getting errors then try Jailbreaking with unc0ver

**Comment by kris0725pl** [comment]
I have hq cable....usb 2.0 only....

I cant turn off passcode....this device have icloud email and password. Not my icloud! This app not working in my opinion. I try wih uncover....

**Comment by DEALWITHIT199** [comment]
ohh the jailbreak will not work if you have your passcode on. well try unc0ver hope it works!

**Comment by DEALWITHIT199** [comment]
unc0ver doesn't need computer ok just sayin

**Comment by kris0725pl** [comment]
ok....uncover is to shit. you dont understand....i cant load uncover because i cant run this iphone...icloud password and email. I dont have password...in my opinion it is not possible crack icloud iphone. every video on youtube is scam for me.

**Comment by kris0725pl** [comment]
i record video on my youtube channel
 and show people true :)

**Comment by DEALWITHIT199** [comment]
> ok....uncover is to shit. you dont understand....i cant load uncover because i cant run this iphone...icloud password and email. I dont have password...in my opinion it is not possible crack icloud iphone. every video on youtube is scam for me.

Don't feel down

**Comment by DEALWITHIT199** [comment]
did you try the AnyUnlock program?

**Comment by kris0725pl** [comment]
Any unlock dont recognize device....u3tools no problem

**Comment by DEALWITHIT199** [comment]
well I'm sorry to say this but I have no more things in mind to help you. hope you find the solution!

**Comment by DEALWITHIT199** [comment]
if unc0ver doesn't work try Odyssey, or Chimera.

**Comment by DEALWITHIT199** [comment]
I get it now. I know why your stuck at "Right before trigger (this is the real bug setup)". Its because iOS 12.4.8 doesn't support checkra1n.

**Comment by DEALWITHIT199** [comment]
> ok....uncover is to shit. you dont understand....i cant load uncover because i cant run this iphone...icloud password and email. I dont have password...in my opinion it is not possible crack icloud iphone. every video on youtube is scam for me.

Unc0ver is the one of the apps that Jailbreaks 12.4.8. Try installing unc0ver to your iDevice using AltStore or if your running a macOS computer use AltDeploy to install unc0ver to you iDevice.

**Comment by demhademha** [comment]
Apple have introduced mitigations designed to prevent icloud bypassing making it extremely difficult.
, 
in addition,, bypassing icloud isn't something the *majority* of the community will  help you with as it raises ethical issues.

**Comment by kris0725pl** [comment]
i cant downgradt from 12.4.8 to 12.4.3....i think it is not possible remove icloud account from iphone 5s when i dont know password and emial.

**Comment by DEALWITHIT199** [comment]
is it iCloud Activation?

**Comment by kris0725pl** [comment]
yes... i cant remove this icloud.

https://ibb.co/47SQsJn

it is not possible

**Comment by DEALWITHIT199** [comment]
ohhhhh. its ez to remove that

**Comment by kris0725pl** [comment]
ez?

**Comment by DEALWITHIT199** [comment]
easy

**Comment by kris0725pl** [comment]
i dont any hope yet...:(

**Comment by DEALWITHIT199** [comment]
go to this link and download the setup trial for windows [https://www.tenorshare.net/products/4mekey-activation-unlocker.html](url)
then after installing the program follow the instructions there.

**Comment by DEALWITHIT199** [comment]
if 4MeKey doesn't work then try this program [https://www.imobie.com/anyunlock/icloud-activation-unlocker.htm](url)
(recommend to try 4MeKey first)

**Comment by kris0725pl** [comment]
https://ibb.co/ZXKHCsp

**Comment by DEALWITHIT199** [comment]
it should be 4MeKey not 4uKey

**Comment by DEALWITHIT199** [comment]
go to this link and download the setup trial for windows https://www.tenorshare.net/products/4mekey-activation-unlocker.html
then after installing the program follow the instructions there.

Note: "Do NOT Jailbreak using 4MeKey" because there is a feature I think Jailbreak so don't jailbreak using 4MeKey. just don't. Jailbreak using bootra1n 0.10.2

**Comment by kris0725pl** [comment]
not working https://ibb.co/7Chhkx1

i cant jailbreak because bootra1n not working! Understand?

**Comment by DEALWITHIT199** [comment]
oh then try continue the jailbreak

**Comment by DEALWITHIT199** [comment]
> not working https://ibb.co/7Chhkx1
> 
> i cant jailbreak because bootra1n not working! Understand?

its not working because of the "Right before trigger (this is the real bug setup)"?

**Comment by kris0725pl** [comment]
yes

**Comment by DEALWITHIT199** [comment]
is the icloud activation lock removed?

**Comment by kris0725pl** [comment]
no

**Comment by DEALWITHIT199** [comment]
try this program. hope it works!
https://www.imobie.com/anyunlock/icloud-activation-unlocker.htm

**Comment by DEALWITHIT199** [comment]
oh wait

**Comment by DEALWITHIT199** [comment]
are you using windows?

**Comment by kris0725pl** [comment]
yes..i dont have mac

**Comment by DEALWITHIT199** [comment]
ok. try the 4meKey again and continue to the step "Please jailbreak your device"

**Comment by DEALWITHIT199** [comment]
ok my time here is 10pm that means it night here so lets continue to talk about this tomorrow. ty

**Comment by kris0725pl** [comment]
i purchase unlock via website...all programs not working.

**Comment by DEALWITHIT199** [comment]
> i purchase unlock via website...all programs not working.

Wdym purchase??? You purchased something??

**Comment by DEALWITHIT199** [comment]
Did 4meKey worked?? dont skip the step “Please jailbreak your device” continue it.

**Comment by DEALWITHIT199** [comment]
Hi. since your iOS iPhone 5s is 12.4.8, watch this [video](https://www.youtube.com/watch?v=2Tq52H1ggng) and try it! hope it work
